gusto/core/configuration.py

```diff
@@ -80,7 +80,11 @@ def __setattr__(self, name, value):
         # Almost all parameters should be Constants -- but there are some
         # specific exceptions which should be kept as integers
         if type(value) in [float, int] and name not in ['dumpfreq', 'pddumpfreq', 'chkptfreq']:
-            object.__setattr__(self, name, Constant(value))
+            object.__setattr__(self, name, value)
+            # DO NOT MERGE
+            # Adjoint is a bit twitchy with Constants so let's not make them
+            # for now
+            # object.__setattr__(self, name, Constant(value))
```

Comment:
Here we should use Function in Real space instead using `Constant`.

Reply:
However, we have to have access to the mesh at this point for using Function in Real Space.